doc-changelog/action.yml

```
@@ -160,23 +160,39 @@ runs:
           rm $file
         fi
 
+    - name: "Clean PR title"
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+      shell: python
+      run: |
+        import os
+
+        # Remove extra whitespace
+        clean_title = " ".join("""${{ env.PR_TITLE }}""".split())
+
+        # Add backslash in front of backtick and double quote
+        clean_title = clean_title.replace("`", "\`").replace('"', '\\"')
```

```suggestion
        clean_title = clean_title.replace("`", "\`").replace('"', '\"')
```

Shouldn't it just be a single backslash?

---

@RobPasMue I think I had to do two backslashes so that the first backslash would escape the second like this:

```
>>> string = input("string: ")
string: hello "world"
>>> print(string.replace('"', '\"'))
hello "world"
>>> print(string.replace('"', '\\"'))
hello \"world\"
```

---

Oh gotcha. You want the backslash to show up in the string. Then yeah